src/controller/python/chip-device-ctrl.py

```diff
@@ -179,6 +180,7 @@ def __init__(self, rendezvousAddr=None, controllerNodeId=0, bluetoothAdapter=Non
         "resolve",
         "zcl",
         "zclread",
+        "zclconfigure",
```

Comment:
Would `zclsubscribe` be more meaningful?

Reply:
when it is aligned towards CHIP data model, yes. for the moment it was aligned with SiLabs ZCL codebase, which uses the term 'configure reporting'.

Reply:
ok!